docs/modules/ROOT/pages/access.adoc

```diff
@@ -138,33 +138,45 @@ const MINTER_ROLE: felt252 = selector!("MINTER_ROLE");
 mod MyContract {
     use openzeppelin::access::accesscontrol::AccessControl as accesscontrol_component;
     use openzeppelin::introspection::src5::SRC5 as src5_component;
-    use openzeppelin::token::erc20::ERC20;
+    use openzeppelin::token::erc20::ERC20 as erc20_component;
     use starknet::ContractAddress;
     use super::MINTER_ROLE;
 
     component!(path: accesscontrol_component, storage: accesscontrol, event: AccessControlEvent);
     component!(path: src5_component, storage: src5, event: SRC5Event);
+    component!(path: erc20_component, storage: erc20, event: ERC20Event);
 
+    // AccessControl
     #[abi(embed_v0)]
     impl AccessControlImpl =
         accesscontrol_component::AccessControlImpl<ContractState>;
     #[abi(embed_v0)]
     impl SRC5Impl = src5_component::SRC5Impl<ContractState>;
```

Comment:
Should we separate this into its own subsection? SRC5 is a requirement for AccessControl, but is not part of it.

```
// SRC5
#[abi(embed_v0)]
impl SRC5Impl = src5_component::SRC5Impl<ContractState>;
```

Reply:
Yes, good call